examples/barchart.html

```diff
@@ -71,6 +76,27 @@ <h1 class="my-5">Bar chart with filters</h1>
                 xAxisId: "age",
                 disaggregateById: "sex",
                 selectedFilterOptions: {}
+            },
+            formatFunction: (value, indicator) => {
```

Comment:
why pass this as a prop rather than have it built into the component? will it ever deviate from this formula?

Reply:
Partly  in order to keep the barchart component generic - maybe we'll have another use for it one day where the formatting requirements are different. Partly so the function we use in HINT only lives in one place, since we use the same method elsewhere in HINT for the tables and maps, and we won't need to publish another barchart version if it needs tweaking.

Reply:
fair enough